src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -722,6 +722,76 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_NoMatchingCertificate" xml:space="preserve">
     <value>The package signature did not match any of the allowed certificate fingerprints.</value>
   </data>
+  <data name="Error_CannotTrustOwnersForAuthor" xml:space="preserve">
+    <value>Trusted owners are not supported in trusted author items.</value>
+  </data>
+  <data name="Error_InvalidCombinationOfArguments" xml:space="preserve">
+    <value>Invalid combination of arguments.</value>
+  </data>
+  <data name="Error_NoSignatureTrustedForPackage" xml:space="preserve">
+    <value>No signature was specified to be trusted for package '{0}'. Please specify either Author or Repository.</value>
```

Comment:
No signature to be trusted was specified for the package

Flip it. The full object before the action.